### ISSUE TITLE
API connection to Office365

### ISSUE THREAD

**Issue by djpirra** [Azure/azure-quickstart-templates#7815]
Is it possible to create a template for deploying an Office 365 connection to use within Logic Apps?

**Comment by MCKLMT** [comment]
Yes, it should be possible to deploy a Logic App from an ARM template. You can check the documentation to understand how to author your script: https://docs.microsoft.com/en-us/azure/logic-apps/logic-apps-azure-resource-manager-templates-overview
However, after deployment, you will have to give access to your tenant from the Logic App user interface to the Office 365 connector.
